pkg/client/cache/listers.go

```diff
@@ -95,6 +95,10 @@ func (s *StoreToPodLister) Exists(pod *api.Pod) (bool, error) {
 	return exists, nil
 }
 
+// NodeConditionPredicate is a function that indicates if a given list of node conditions match the
```

Comment:
Done.

Thanks for that wording. I was having trouble to make this comment more readable.